openerp/addons/base/i18n/base.pot

```diff
@@ -10041,6 +10299,18 @@ msgid "Missing SMTP Server"
 msgstr ""
 
 #. module: base
+#: code:addons/models.py:6036
+#, python-format
+msgid "Missing required value for the field '%s' (%s)"
```

Comment:
Using positional placeholders make it more difficult to translators to translate the string.
For 2 reasons:
 - You don't know the what's it's refer to
 - The placeholders may have to be inverted in some languages.